diagnostic.md

```diff
@@ -140,7 +175,9 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne will be on the upper row on the right side.
+divTwo and divThree will be in the middle row (side by side).
+divFour will be on the final row (because it cleared it's float). 
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree)